inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -740,69 +767,91 @@ linelist_cleaned <- linelist_cleaned %>%
  #      No  = "no",  
  #      No  = "0",  
  #    )     
+
+
+
+# Create a variable based on rules from other simple character variables
+# If you have access to lab results, you can create a case definition variable 
+# the tilda (~) is used to assign the new values (Conf, prob, susp, unknown)
+# starting from the specific to the general
+# TRUE assigns all remaining rows 
+# You MUST modify this section to match your case definition. The below
+# uses positive RDT for Confirmed and epi link only for Probable.
+#
+linelist_cleaned <- linelist_cleaned %>%
+  mutate(case_def = case_when(
+    is.na(hep_e_rdt) & is.na(other_cases_in_hh)           ~ NA_character_,
+    hep_e_rdt == "Positive"                               ~ "Confirmed",
+    hep_e_rdt != "Positive" & other_cases_in_hh == "Yes"  ~ "Probable",
+    TRUE                                                  ~ "Suspected"
+  ))
+```
   
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+factor_vars
+
+This chunk will help you clean factor variables.
```

Comment:
summary of different operations?

Reply:
i like leaving it in the chunk as comments -- not sure people will read or use more details

Reply:
cool